app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user = User.first
+  end
 end
```

Comment:
- [ ] Let's also add comment `controllers` and `likes` controllers. Where comment is  created on behalf of the currently logged in user (current_user) of the app. A user should be able to add a like to posts.
- [ ] I recommend you to customize your readme file to match the current project contents. you can follow this [template.](https://github.com/microverseinc/readme-template/blob/master/README.md)